CONTRIBUTION_GUIDELINES.md

```diff
@@ -114,6 +114,15 @@ below.
       by the issue number. This allows GitHub to automatically link and close
       issues when the PR is merged.
 
+   - **NOTE** In _docs_ directory a _commit-template_ file is
+present. You can just copy this file in your home folder (maybe as a hidden
+file) and then add the following lines to your .gitconfig:
```

Comment:
I'd prefer this:

NOTE: A commit-template file is available in the docs directory.
You can copy it to your home directory (optionally as a hidden file) and add the following to your .gitconfig: